server/src/main/java/org/room/apollo/server/service/AuthorizationService.java

```diff
@@ -0,0 +1,72 @@
+package org.room.apollo.server.service;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.room.apollo.server.configuration.DeezerConfiguration;
+import org.room.apollo.server.dto.deezer.DeezerToken;
+import org.room.apollo.server.util.HttpUtil;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Service;
+
+import java.io.IOException;
+import java.net.URI;
+import java.net.URISyntaxException;
+
+/**
+ * Service for authorization methods.
+ */
+@Service
+public class AuthorizationService {
+
+    private DeezerConfiguration deezerConfiguration;
```

Comment:
Can we autowire (inject) values of DeezerConfiguration and HttpUtil?

Reply:
We are Autowiring through constructor.